preprocess/raven_preprocess/test_dataverse.py

```diff
@@ -1,6 +1,7 @@
 import csv
 import glob
 import json
+import pycountry
```

Comment:
Note: add pycountry to the pip requirements file: 
  -  10_preprocess.txt
  (can rename reqs file at later date...)